README.md

```diff
@@ -204,44 +204,62 @@ Prior to cloning the repository and attempting to build the project, ensure that
 To use test datasets you will need:
 - Git Large File Storage, version 3.0.2+
 
-### Building the project (first option: with tests)
+### Building the project
+#### Building the Python module using pip (this is what you probably want, if you're not a developer)
 Firstly, navigate to a desired directory.
-Then, clone the repository, cd into the project directory and launch the build script:
+Then, clone the repository, cd into the project directory and launch the following commands:
 ```
-git clone https://github.com/Mstrutov/Desbordante/
-cd Desbordante
-./pull_datasets.sh
-./build.sh
+python3 -m venv venv
+source venv/bin/activate
+python3 -m pip install .
 ```
 
-### Building the project (second option: without tests)
+Now it's possible to `import desbordante` as a module from within created virtual environment. In particular, now CLI can be used:
```

Comment:
1) it's -> it is
2) In particular... -> Also, this step is mandatory for setting up the CLI interface of Desbordante.